app/services/most_popular_content.rb

```diff
@@ -0,0 +1,35 @@
+class MostPopularContent
+  attr_reader :content_id
+
+  def initialize(content_id:)
+    @content_id = content_id
+  end
+
+  def self.fetch(content_id:)
+    new(content_id: content_id).fetch
+  end
+
+  def fetch
+    search_response
+      .documents
+      .sort_by { |document| document.title }
+  end
+
+private
+
+  def hide_from_most_popular
```

Comment:
There's a small chance that - if this does indeed come up in the most popular - then we'll get a list of 6 items instead of 7. I don't mind that much, but we may want to check with Michael/Alex if this is okay?